app/models/rental.py

```diff
@@ -0,0 +1,26 @@
+from flask import current_app
+from sqlalchemy.sql.schema import ForeignKey
+from app import db
+from sqlalchemy.orm import relationship
+from datetime import datetime
+# from sqlalchemy import Table, Column, Integer, ForeignKey # Need?
+
+class Rental(db.Model):
+    __tablename__ = "rentals"
+
+    rental_id =db.Column(db.Integer, primary_key=True, autoincrement=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('videos.video_id'),nullable=True)
+    customer_id = db.Column(db.Integer, ForeignKey("customers.customer_id"),primary_key=True)
+    due_date = db.Column(db.DateTime, nullable=True)
+    customer_id = db.Column(db.Integer, ForeignKey("customers.customer_id"),primary_key=True)
```

Comment:
Instead of importing things from `sqlalchemy` you should just access them through `db`:

```suggestion
from app import db
from sqlalchemy.orm import relationship

class Rental(db.Model):
    __tablename__ = "rentals"

    rental_id =db.Column(db.Integer, primary_key=True, autoincrement=True)
    video_id = db.Column(db.Integer, db.ForeignKey('videos.video_id'),nullable=True)
    customer_id = db.Column(db.Integer, db.ForeignKey("customers.customer_id"),primary_key=True)
    due_date = db.Column(db.DateTime, nullable=True)
    customer_id = db.Column(db.Integer, db.ForeignKey("customers.customer_id"),primary_key=True)
```

You also don't need to import `datetime` since it's different from `db.DateTime`.